docs/src/main/tut/auth.md

```diff
@@ -25,25 +25,22 @@ import scalaj.http.HttpResponse
 
 ```
 
-They also make use of `cats.Id` but any type container implementing `MonadError[M, Throwable]` will
-do.
-
 ## Authentication
 
 ### Create a new authorization token
 
-Used to request a new auth token given basic authentication
+Used to request a new auth token given basic authentication.
 
-You can create a tree using `newAuth`, it takes as arguments:
+You can create a new authorization token using `newAuth`, it takes as arguments:
 
-- basic authentication for the token holder (`username` and `password`)
-- `scopes`: attached to the token, for more information see [the scopes doc](https://developer.github.com/v3/oauth/#scopes)
-- `note`: to remind you what the OAuth token is for
-- `client_id`: the 20 character OAuth app client key for which to create the token
-- `client_secret`: the 40 character OAuth app client secret for which to create the token
+- basic authentication for the token holder (`username` and `password`).
+- `scopes`: attached to the token, for more information see [the scopes doc](https://developer.github.com/v3/oauth/#scopes).
+- `note`: to remind you what the OAuth token is for.
+- `client_id`: the 20 character OAuth app client key for which to create the token.
+- `client_secret`: the 40 character OAuth app client secret for which to create the token.
 
 ```scala
-val newAuth = Github(accessToken).auth.newAuth(
+val newAuth = Github(None).auth.newAuth(
```

Comment:
It's `None` because you don't have any accessToken yet, anyway i'm going to document it